scripts/find_pulsar_in_obs.py

```diff
@@ -690,10 +697,10 @@ def write_output_source_files(output_data,
             else:
                 output_file.write('\n')
             for data in output_data[source]:
-                obsid, duration, enter, exit, max_power = data
+                obsid, duration, enter, exit, max_power, freq, band = data
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'exit'](https://app.codacy.com/manual/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4896989)